meetings/2022-11-16.md

```diff
@@ -45,45 +57,103 @@
+* DB: Last time in the WebID meeting I remember Tim saying he was interested in having the possibility of a minimum profile. Name, pictures, things apps would consume in a daily basis. Reduce amount of hops for this popular information. Could there be a distinction of priorities in data? 
+* SC: That's an old design pattern that has been in Solid. Just to make it useful. We're talking about Social linked data, the basics, even if it's a pseudonym, it puts something on a screen. If people want to put their real photo in a different location, that's fine. There's a distinction between making some info available for the utility of it, not so much exposing it. You can have anonymous and pseudonymous WebIDs. 
+* WT: I agree that it is important that apps can access important information quickly. But I'd like to raise: 1 - it is quite naive to think that we know what apps will need now or in the future. Even if we decide it's a good idea to have these things, I think they should be in there using some mechanism that relies on the standard discoverable place where they are. You can imagine we store all data even names and avatars in the interop registries, but we let the authz agent/server of interop make it so that if you declare that a certain type of data is public, then ??? is added to your WebID profile. Just brainstorming. 
+* HS: I think we should also be careful about premature optimization in this case. The argument that you following links might be problematic because it might lead you nowhere, that's something you can deal with but expensive. You can have a version 2 of Solid that allows you to specify filters, it could be really complicated, because we're working on consensus... ??? we def could do that, but it's going to take a long time and resources to build. If we look at Mastodon, it's got public profiles. You can put your picture up... so we def wanna get to the point of military grade security but let's avoid tying ourselves into knots. Let's not build again security, that's it. 
+* SC: We do have an understanding that if we can have a solution that you can extend to a point where you can have the basic thing now and leave enough space to solve that part later... without breaking changes... sometimes we're looking for a balance.
+* eP: +1 to avoiding premature optimization. We need some grounds before we start improving.
+* eP: We need to work based on combined use cases, I think about social graph use cases, because then we have authz and security considerations we have address book and agent registry in SAI and we need a consistent way of working and presenting information. We need for example to prevent phishing because of people impersonating others. when it comes to social graph, There are some issues on github related to working with info about individuals, specially on contacts later used for authorization.
```

Comment:
```suggestion
* eP: We need to work based on combined use cases. I think about social graph use cases, because then we have authz and security considerations. We have address book and agent registry in SAI, and we need a consistent way of working and presenting information. We need, for example, to prevent phishing because of people impersonating others. When it comes to social graph, There are some issues on github related to working with info about individuals, especially on contacts later used for authorization.
```